src/types.ts

```diff
@@ -154,20 +154,23 @@ export type JobsSearchResult = MiscTypes.BaseSearchResult & {
 }
 
 export interface FormMigrationOptions {
-  /** formsAppEnvironmentId is required. The Forms App Environment to migrate to  */
+  /** FormsAppEnvironmentId is required. The Forms App Environment to migrate to */
```

Comment:
Leave these as the name of the property. Without capitalisation. could cause confusion.